tests/StereoDB.Tests/HeartbeatTests.fs

```diff
@@ -14,25 +14,30 @@ open Tests.TestHelper
 open System
 open System.Threading
 
-[<Fact>]
+[<Fact(Skip = "requires local kafka")>]
```

Comment:
Maybe we can mark with using category, and document command how we can run with test category disabled?

Reply:
yep, agree that were my initial thoughts, updated in recent commit
https://github.com/StereoDB/StereoDB/pull/45#issuecomment-2558068254